packages/x-data-grid/src/constants/cssVariables.ts

```diff
@@ -0,0 +1,168 @@
+// NOTE: Breakpoints can't come from the theme because we need access to them at
+// initialization time and media-queries can't use CSS variables. For users with
+// custom breakpoints, we might want to provide a way to configure them globally
+// instead of through the theme.
+const breakpoints = {
+  values: {
+    xs: 0, // phone
+    sm: 600, // tablet
+    md: 900, // small laptop
+    lg: 1200, // desktop
+    xl: 1536, // large screen
+  },
```

Comment:
We can't have this as far as I know. This is Material UI ownership.

```suggestion
```

Reply:
The comment above explains the issue, but CSS variables can't be used in media queries, so we need access to the breakpoints as soon as we declare the (now static) styles to support the `[t.breakpoints.up('sm')]: { ... }` syntax.

Reply:
My thought was that we don't need breakpoints in the first place. Assuming that the style in the "unstyled" version of the data grid should be as minimal as possible, breakpoint aren't needed to guarantee layout consistency.